src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag

```diff
@@ -9,216 +9,216 @@
+                          <li class="list-group-item">Unpublish Evaluation</li>
+                          <li class="list-group-item">Send Registration</li>
+                        </ul>
+                      </td>
+
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Pageload</li>
+                          <li class="list-group-item">System Error Report</li>
+                          <li class="list-group-item">Servlet Action Failure</li>
+                        </ul>
+                      </td>
+                    </tr>
+
+                    <tr>
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Add New Course</li>
+                          <li class="list-group-item">Delete Course</li>
+                          <li class="list-group-item">Edit Course Info</li>
+                        </ul>
+                      </td>
+
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Enroll Students</li>
+                          <li class="list-group-item">Edit Student Details</li>
+                          <li class="list-group-item">Delete Student</li>
+                          <li class="list-group-item">Student Joining Course</li>
+                        </ul>
+                      </td>
+
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Create New Evaluation</li>
+                          <li class="list-group-item">Edit Evaluation Info</li>
+                          <li class="list-group-item">Delete Evaluation</li>
+                          <li class="list-group-item">Edit Submission</li>
+                        </ul>
+                      </td>
+                    </tr>
+                  </table>
+                </div>
+              </p>
+            </div>
+          </div>
+        </small>
+      </div>
+
+    </div>
+
+    <input type="hidden" name="pageChange" value="true">
+
+    <%-- This parameter determines whether the logs with requests contained in "excludedLogRequestURIs"
+    in AdminActivityLogPageData should be shown. Use "?all=true" in URL to show all logs. This will keep showing all
+    logs despite any action or change in the page unless the page is reloaded with "?all=false"
+    or simply reloaded with this parameter omitted. --%>
+    <input type="hidden" name="all" value="${shouldShowAllLogs}">
+
+    <%-- This determines whether the logs related to testing data should be shown. Use "testdata=true" in URL
+    to show all testing logs. This will keep showing all logs from testing data despite any action or change in the page
+    unless the page is reloaded with "?testdata=false"  or simply reloaded with this parameter omitted. --%>
```

Comment:
In addition to the multiline comment convention, remove the extra whitespace between `"?testdata=false"` and `or`.